### ISSUE TITLE
Disable conversion of confidence summation to 1

### ISSUE THREAD

**Issue by fruworg** [ultralytics/ultralytics#14659]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi there. When I try to classify an object using yolov8, the total confidence value is always 1. This results in a false positive classification. How can I get the confidence for each class?

What I have: `0: 640x640 CAT 0.66, DOG 0.32, BIRD 0.01, FISH 0.01, TURTLE 0.00, 25.1ms`
What I want to get: `0: 640x640 CAT 0.80, DOG 0.73, BIRD 0.20, FISH 0.17, TURTLE 0.00, 25.1ms`
What I want to get (most likely): `0: 640x640 CAT 0.19, DOG 0.18, BIRD 0.03, FISH 0.01, TURTLE 0.00, 25.1ms`

(btw, I'm using a custom dataset)

**Comment by Y-T-G** [comment]
You will have to change the softmax to sigmoid.

https://github.com/ultralytics/ultralytics/blob/9df18ca9eff114743e3752ff95fc9c35d8e1202b/ultralytics/nn/modules/head.py#L280

**Comment by fruworg** [comment]
@Y-T-G, thank you

**Comment by fruworg** [comment]
Hi, @Y-T-G
I checked again and it didn't help. The total is still 1.

#### yolo_sigmoid/ultralytics/nn/modules/head.py
```diff
- return x if self.training else x.softmax(1)
+ return x if self.training else x.sigmoid(1)
```

#### yolo entrypoint
```python
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from yolo_sigmoid.ultralytics.cfg import entrypoint
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(entrypoint())

```

#### train & predict
```shell
./yolo classify train data=/opt/yolo/custom model=yolov8-cls.yaml epochs=50 imgsz=128
./yolo classify predict model=/opt/yolo/runs/classify/train/weights/best.pt source=/tmp/image.jpg
```

#### result
```shell
0: 640x640 CAT 0.69, DOG 0.27, BIRD 0.03, FISH 0.01, TURTLE 0.00, 25.1ms
```

**Comment by Y-T-G** [comment]
Did you modify it correctly and install the updated package? I don't think so, because `x.sigmoid(1)` will throw an error of the changes took effect because that's not the correct syntax. It should be `torch.sigmoid(x,)`.

**Comment by fruworg** [comment]
@Y-T-G, yeah, you were right. Thanks!
